docs/src/main/sphinx/connector/delta-lake.rst

```diff
@@ -707,6 +707,28 @@ The output of the query has the following history columns:
     - ``boolean``
     - Whether or not the operation appended data
 
+``$properties`` table
+~~~~~~~~~~~~~~~~~~~~~
+
+The ``$properties`` table provides access to Delta table configuration,
+table features and table properties. The table rows are key/value pairs.
+
+You can retrieve the properties of the Delta
+table ``test_table`` by using the following query::
+
+    SELECT * FROM "test_table$properties"
+
+.. code-block:: text
+
+     key                        | value           |
+    ----------------------------+-----------------+
+    someStringProperty          | someStringValue |
```

Comment:
pls add a compatibility test for dealing with case sensitive property names and values

Reply:
Added a `'foo' = 'bar', 'Foo' = 'Baz'` case to the product test. `foo` and `Foo` are distinct property names.